[2025-10-13T08:41:45.017+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T08:41:45.043+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-13T08:41:45.055+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-13T08:41:45.056+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T08:41:45.073+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-09-04 00:00:00+00:00
[2025-10-13T08:41:45.080+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=353) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T08:41:45.083+0000] {standard_task_runner.py:72} INFO - Started process 360 to run task
[2025-10-13T08:41:45.083+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp1dz_x9no']
[2025-10-13T08:41:45.085+0000] {standard_task_runner.py:105} INFO - Job 53: Subtask extract_load_group.load_customers
[2025-10-13T08:41:45.153+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T08:41:45.256+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-10-13T08:41:45.257+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T08:41:56.809+0000] {load.py:30} INFO - Loaded 0 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-13T08:41:56.810+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T08:41:56.823+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T08:41:56.823+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20251013T084145, end_date=20251013T084156
[2025-10-13T08:41:56.886+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T08:41:56.904+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T08:41:56.907+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-13T09:21:30.799+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-13T09:21:30.820+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-13T09:21:30.830+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-13T09:21:30.831+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-13T09:21:30.844+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-09-04 00:00:00+00:00
[2025-10-13T09:21:30.852+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1418) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-13T09:21:30.854+0000] {standard_task_runner.py:72} INFO - Started process 1419 to run task
[2025-10-13T09:21:30.854+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '202', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpc1mst2jy']
[2025-10-13T09:21:30.855+0000] {standard_task_runner.py:105} INFO - Job 202: Subtask extract_load_group.load_customers
[2025-10-13T09:21:30.914+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [running]> on host 84d5a327cf64
[2025-10-13T09:21:31.006+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-10-13T09:21:31.007+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-13T09:21:39.817+0000] {load.py:30} INFO - Loaded 25 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-13T09:21:39.817+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-13T09:21:39.829+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-13T09:21:39.830+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20251013T092130, end_date=20251013T092139
[2025-10-13T09:21:39.866+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-13T09:21:39.888+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-13T09:21:39.890+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:21:27.413+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:21:27.442+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:21:27.454+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:21:27.454+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:21:27.474+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-09-04 00:00:00+00:00
[2025-10-14T04:21:27.488+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=408) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:21:27.490+0000] {standard_task_runner.py:72} INFO - Started process 420 to run task
[2025-10-14T04:21:27.491+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpdsudf8m5']
[2025-10-14T04:21:27.493+0000] {standard_task_runner.py:105} INFO - Job 303: Subtask extract_load_group.load_customers
[2025-10-14T04:21:27.558+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:21:27.656+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-10-14T04:21:27.658+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:21:31.768+0000] {load.py:30} INFO - Loaded 25 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-14T04:21:31.769+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:21:31.782+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:21:31.783+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20251014T042127, end_date=20251014T042131
[2025-10-14T04:21:31.843+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:21:31.869+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:21:31.872+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:29:28.561+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:29:28.602+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:29:28.618+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:29:28.622+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:29:28.647+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-09-04 00:00:00+00:00
[2025-10-14T04:29:28.656+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=902) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:29:28.658+0000] {standard_task_runner.py:72} INFO - Started process 907 to run task
[2025-10-14T04:29:28.658+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmp_0861n0x']
[2025-10-14T04:29:28.660+0000] {standard_task_runner.py:105} INFO - Job 408: Subtask extract_load_group.load_customers
[2025-10-14T04:29:28.749+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:29:28.914+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-10-14T04:29:28.915+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:29:32.765+0000] {load.py:30} INFO - Loaded 25 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-14T04:29:32.765+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:29:32.780+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:29:32.780+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20251014T042928, end_date=20251014T042932
[2025-10-14T04:29:32.850+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:29:32.864+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-10-14T04:36:24.507+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-14T04:36:24.533+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:36:24.543+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [queued]>
[2025-10-14T04:36:24.543+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-10-14T04:36:24.564+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_load_group.load_customers> on 2025-09-04 00:00:00+00:00
[2025-10-14T04:36:24.573+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1251) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-14T04:36:24.575+0000] {standard_task_runner.py:72} INFO - Started process 1260 to run task
[2025-10-14T04:36:24.575+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'elt-pipeline', 'extract_load_group.load_customers', 'scheduled__2025-09-04T00:00:00+00:00', '--job-id', '482', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpp5bxtm7j']
[2025-10-14T04:36:24.576+0000] {standard_task_runner.py:105} INFO - Job 482: Subtask extract_load_group.load_customers
[2025-10-14T04:36:24.645+0000] {task_command.py:467} INFO - Running <TaskInstance: elt-pipeline.extract_load_group.load_customers scheduled__2025-09-04T00:00:00+00:00 [running]> on host e348b991e0be
[2025-10-14T04:36:24.756+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt-pipeline' AIRFLOW_CTX_TASK_ID='extract_load_group.load_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-04T00:00:00+00:00'
[2025-10-14T04:36:24.757+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-14T04:36:28.751+0000] {load.py:30} INFO - Loaded 25 rows and 5 columns to gcpl-469311.datalabs_techtest.stg_customers
[2025-10-14T04:36:28.752+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-10-14T04:36:28.768+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-14T04:36:28.769+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=elt-pipeline, task_id=extract_load_group.load_customers, run_id=scheduled__2025-09-04T00:00:00+00:00, execution_date=20250904T000000, start_date=20251014T043624, end_date=20251014T043628
[2025-10-14T04:36:28.804+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-14T04:36:28.831+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-14T04:36:28.833+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
